manage.py

```diff
@@ -195,7 +195,7 @@ def refresh_itemized():
     """These are run nightly to refresh the itemized schedule A and B data."""
 
     refresh_itemized_a()
-    refresh_itemized_b()
+    # refresh_itemized_b()
```

Comment:
Why not just remove the line instead of commenting it out? Same comment applies to other places as well.